modules/caddyhttp/fileserver/browse.html

```diff
@@ -790,6 +790,9 @@ <h1>
 						<span class="meta-item">
 							<b>{{.NumFiles}}</b> file{{if ne 1 .NumFiles}}s{{end}}
 						</span>
+						<span class="meta-item">
+							<b>{{.HumanTotalFileSize}}</b> in total 
```

Comment:
```suggestion
							<b>{{.HumanTotalFileSize}}</b> total 
```

(This is just a nit, and maybe "in total" is better grammar, but I am not sure.)